pkg/util/util.go

```diff
@@ -430,13 +430,13 @@ func ResolvePVCFromPod(pod *corev1.Pod, pvcLister corelisterv1.PersistentVolumeC
 			pvc, err := pvcLister.PersistentVolumeClaims(pod.Namespace).Get(pvcName)
 			if err != nil {
 				klog.Errorf("Get PVC %s/%s error: %v", pod.Namespace, pvcName, err)
-				continue
+				return nil, err
 			}
 			pvcs = append(pvcs, pvc)
 		}
 	}
 	if len(pvcs) == 0 {
-		return nil, errors.NewNotFound(corev1.Resource("pvc"), pod.Name)
+		return nil, fmt.Errorf("no pvc found for pod %s/%s", pod.Namespace, pod.Name)
```

Comment:
Why return the error here?
For this function, the Pod may include PVC or not, both cases are possible behavior, and the caller should decide whether `len(pvcs) == 0` is expected or not and handle this case.

Reply:
fixed.